base/rational.jl

```diff
@@ -98,9 +98,34 @@ function //(x::Rational, y::Rational)
 end
 
 //(x::Complex, y::Real) = complex(real(x)//y, imag(x)//y)
-//(x::Number, y::Complex) = x*conj(y)//abs2(y)
-
-
+function //(x::Number, y::Complex)
+    if((x//abs2(y))==0//1 || (x//abs2(y))==1//0)
```

Comment:
Comparing with `iszero`, `isinf`, etc., should be cheaper than with `==`.

Reply:
implemented!